src/Microsoft.Framework.DependencyInjection.Abstractions/project.json

```diff
@@ -8,7 +8,7 @@
         "net45": {
         },
         "dnx451": { },
```

Comment:
Is there (and if not, can there be) any sort of public notes on the tfn's/monikers/compat matrix? I.e., what's this supposed to mean w.r.t. netcore*, net*, etc?

Thanks!